From fb8f7e8aa0c9b93b554abc56a15483f0f54a4006 Mon Sep 17 00:00:00 2001
From: Victor Wan <victor.wan@amlogic.com>
Date: Fri, 4 Jul 2014 19:23:58 +0800
Subject: [PATCH 4478/5965] m8m2: cpu: fix unique chip-id

m8m2 HW chip-id same as m8 (25,0x19), this patch change this value to (29,0x1d)
---
 arch/arm/mach-meson8/cpu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-meson8/cpu.c b/arch/arm/mach-meson8/cpu.c
index 58ec15da5d70..14c9f2a5dbf1 100755
--- a/arch/arm/mach-meson8/cpu.c
+++ b/arch/arm/mach-meson8/cpu.c
@@ -41,6 +41,8 @@ int __init meson_cpu_version_init(void)
 	
 	version = aml_read_reg32(P_METAL_REVISION);
 	switch (version) {		
+		case 0x11111112:
+			meson_cpu_version[MESON_CPU_VERSION_LVL_MAJOR] = 0x1D; //M8 MarkII RevA
 		case 0x11111111:			
 			ver = 0xA;		
 			break;		
-- 
2.19.0

